CHANGELOG.next.asciidoc

```diff
@@ -151,6 +151,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix terminating pod autodiscover issue. {pull}20084[20084]
 - Fix seccomp policy for calls to `chmod` and `chown`. {pull}20054[20054]
 - Output errors when Kibana index pattern setup fails. {pull}20121[20121]
+- [Autodiscovery] Ignore ErrInputNotFinished errors in autodiscover config checks. {pull}20305[20305]
```

Comment:
Rephrase this to something that indicates solved user-facing issue, e.g, something like:
```suggestion
- Fix issue in autodiscover that kept inputs stopped after config updates. {pull}20305[20305]
```